platform/android/gradle/gradle-javadoc.gradle

```diff
@@ -14,5 +14,6 @@ android.libraryVariants.all { variant ->
         options.overview("src/main/java/overview.html")
         options.group("Mapbox Android SDK", "com.mapbox.*")
         exclude '**/R.java', '**/BuildConfig.java', '**/**.java.ejs', '**/**.html'
+        failOnError false
```

Comment:
Thanks for looking into this. I think for now this is fine as the warning existed before already I think. Can you isolate this change and make a pull request with only this file here?

Reply:
Sure